woocommerce/woocommerce-ios
# Add swap camera button to code scanner view controller

## Description


In current code scanner user can only use back camera. With this PR changes there is a **swap camera** button added on top right and by tapping it we swap/switch between front and back camera.
Of course, if one of those cameras is not available we hide the button to avoid bad experience with having a button that does nothing.

## Testing information


- Run the app on the physical device, iPhone or iPad
- Open Products tab
- Tap the scan button in top left
- Check that the swap camera button is shown in the code scanner
- Tap the swap camera button few times and make sure that the camera is actually being swapped and you are getting proper camera feed

## Screenshots


https://github.com/user-attachments/assets/f7ab02b8-9c61-48a1-9961-5eb3369b2605

---

## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:
